src/algorithms/depminer/CMAXSet.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <unordered_set>
+
+#include "ColumnLayoutRelationData.h"
+#include "custom/CustomHashes.h"
+
+class CMAXSet{
+private:
+    Column column;
+    std::unordered_set<Vertical> columnCombinations;
+public:
+    CMAXSet(Column const& column) : column(column) {};
```

Comment:
`explicit CMAXSet` -- Single-argument constructors must be marked explicit to avoid unintentional implicit conversions